src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -30,10 +30,23 @@ CaptureOptionsDialog::CaptureOptionsDialog(QWidget* parent)
 
   QObject::connect(ui_->buttonBox, &QDialogButtonBox::accepted, this, &QDialog::accept);
   QObject::connect(ui_->buttonBox, &QDialogButtonBox::rejected, this, &QDialog::reject);
+  QObject::connect(ui_->unwindingMethodComboBox, qOverload<int>(&QComboBox::currentIndexChanged),
+                   ui_->maxCopyRawStackSizeLineEdit, [this]() {
+                     auto unwinding_method = ui_->unwindingMethodComboBox->currentData().toInt();
+                     ui_->maxCopyRawStackSizeLineEdit->setEnabled(unwinding_method ==
+                                                                  CaptureOptions::kFramePointers);
```

Comment:
When DWARF is selected, this is disabled. But from the screenshot, if one has only used DWARF, they could think that 512 is the value and it cannot be changed. I don't know if this is actually a problem, maybe it's just my feeling right now. But in case, I propose one of the following:
- Hide the option completely when DWARF is selected, instead of disabling it;
- Add "(frame pointers only)" to the text.

What do you think?